docs/docs/centralizing-your-sites-navigation.md

```diff
@@ -195,62 +195,64 @@ Locate the `header.js` file inside `src/components` and remove everything so onl
 import React from 'react'
 import { Link } from 'gatsby'
 const Header = ({ siteTitle, menuLinks }) => (
--  <div
--    style={{
--      background: 'rebeccapurple',
--      marginBottom: '1.45rem',
--    }}
--  >
--    <div
--      style={{
--        margin: '0 auto',
--        maxWidth: 960,
--        padding: '1.45rem 1.0875rem',
--      }}
--    >
--      <div style={{
--        display: 'flex',
--        'justifyItems': 'space-between',
--        'alignItems': 'center'
--      }}>
--        <h1 style={{ margin: 0, flex: 1 }}>
--          <Link
--            to="/"
--            style={{
--              color: 'white',
--              textDecoration: 'none',
--            }}
--          >
--            {siteTitle}
--          </Link>
--        </h1>
--
--      </div>
--    </div>
--  </div>
+  <header
+    style={{
+      background: 'rebeccapurple',
+      marginBottom: '1.45rem',
+    }}
+  >
+    <div
+      style={{
+        margin: '0 auto',
+        maxWidth: 960,
+        padding: '1.45rem 1.0875rem',
++        display: 'flex',
++        'justifyItems': 'space-between',
++        'alignItems': 'center'
+      }}
+     >
+        <h1 style={{ margin: 0, flex: 1 }}>
+          <Link
+            to="/"
+            style={{
+              color: 'white',
+              textDecoration: 'none',
+            }}
+          >
+            {siteTitle}
+          </Link>
+        </h1>
++        <div>
++					<nav style={{ display: 'flex', flex: 1 }}>
```

Comment:
We need to incorporate a `<ul>` element here to make the list items into proper HTML. Here is a suggested structure:
```
<nav>
    <ul style={{ display: 'flex', flex: 1 }}>
        { menuLinks.map(link => 
            <li key={link.name} ...></li>
        }
    </ul>
</nav>
```
By putting a `<ul>` inside the nav and moving the `display: flex` there, the list item should still be flex children.